src/test/cbmc/DynTrait/dyn_fn_mut_fail.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Check that we can pass a dyn FnMut pointer to a stand alone
+// function definition. Fail checks the wrong int.
```

Comment:
```suggestion
// function definition. Fails when comparing with a non-expected value.
```
Or something similar.